README.md

```diff
@@ -120,25 +125,30 @@ $ git clone https://github.com/softdevteam/krun.git
 $ cd krun
 ```
 
-## Step 3: set `JAVA_HOME`
+## Step 3: Build Krun
+
+The Krun Makefile honours the standard variables: `CC`, `CPPFLAGS`, `CFLAGS`
+and `LDFLAGS`. For example, if you wish to use `clang` rather than `gcc` you
+can append `CC=/bin/clang` to the `make` command here:
 
 ```bash
-$ export JAVA_HOME=/usr/lib/jvm/java-7-openjdk-amd64/
+$ pwd
+.../Krun
+$ make
```

Comment:
All `make` would need to be `gmake` on most non-linuxes. Maybe a quick note near the top. The user needs to invoke GNU make.